packages/react/tsconfig.json

```diff
@@ -1,17 +1,10 @@
 {
+  "extends": "../configs/ts/react.json",
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
-    "declaration": true,
-    "esModuleInterop": true,
-    "importHelpers": true,
-    "jsx": "react",
-    "module": "esnext",
-    "moduleResolution": "node",
-    "rootDir": "./src",
+    "noImplicitAny": false,
     "skipLibCheck": true,
```

Comment:
Added `skipLibCheck` back as removing exposed a linting error caused by a type error in _maplibre-gl-js-amplify_:
```sh
$ yarn workspace @aws-amplify/ui-react lint
$ tsc --noEmit && eslint src --ext .js,.ts,.tsx
../../node_modules/maplibre-gl-js-amplify/lib/esm/AmplifyGeofenceControl/AmplifyMapDraw.d.ts:1:23 - error TS2688: Cannot find type definition file for 'mapbox__mapbox-gl-draw'.

1 /// <reference types="mapbox__mapbox-gl-draw" />
                        ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in ../../node_modules/maplibre-gl-js-amplify/lib/esm/AmplifyGeofenceControl/AmplifyMapDraw.d.ts:1
```

Will add a github issue there to address it and can remove the flag once it's fixed.